tests/app/test_queue.py

```diff
@@ -117,46 +116,46 @@ def test_polling_message_with_custom_inbox_name(self, redis, suffix):
             (receipt, elements) = redis_queue.poll(10)
             assert len(elements) == 1
             assert redis.llen(Buffer.INBOX.name(suffix)) == 0
-            assert redis.llen(redis_queue.get_inflight_name(receipt)) == 1
+            assert redis.llen(Buffer.IN_FLIGHT.get_inflight_name(receipt, suffix)) == 1
 
             redis_queue.acknowledge(receipt)
             assert redis.llen(Buffer.INBOX.name(suffix)) == 0
-            assert redis.llen(redis_queue.get_inflight_name(receipt)) == 0
+            assert redis.llen(Buffer.IN_FLIGHT.get_inflight_name(receipt, suffix)) == 0
         finally:
             self.delete_all_list(redis)
 
     @pytest.mark.serial
-    def test_polling_with_empty_inbox(self, redis, redis_queue):
+    def test_polling_with_empty_inbox(self, app, redis, redis_queue):
```

Comment:
This is not necessary to define the `app` mock as a parameter because of the `autouse=True` usage [at this line](https://github.com/cds-snc/notification-api/pull/1451/files#diff-c9d36498371981109d1bf2de7b12bb84964b6f4a6b55a03af7bf94ca02781366R22).

Reply:
An example would be helpful.?